File: blurview/src/main/java/com/devgxy/blur/ToolKitBlurImpl.java
package com.devgxy.blur;

import android.graphics.Bitmap;

import com.google.android.renderscript.Toolkit;


public class ToolKitBlurImpl implements IBlur {
    float radius = 1f;

    @Override
    public void release() {

    }

    @Override
    public void blur(Bitmap input, Bitmap output) {
        Toolkit.INSTANCE.blur(input, radius, output);
    }
}

File: blurview/src/main/java/com/devgxy/blur/ToolKitBlurImpl.kt
package com.devgxy.blur

import android.graphics.Bitmap
import com.google.android.renderscript.Toolkit.blur

/**
 * 使用ToolKit实现模糊功能
 */
class ToolKitBlurImpl : IBlur {
    override var radius: Float = 1f

    override fun release() {
    }

    override fun blur(input: Bitmap, output: Bitmap) {
        blur(input, radius, outputBitmap = output)
    }
}
